sidebars.js

```diff
@@ -52,7 +52,16 @@ module.exports = {
         {
           type: 'category',
           label: 'Experiments 🧪',
-          items: ['experiments/overview', 'experiments/generic-pattern-matching', 'experiments/join-mode', 'experiments/project-depends-on', 'experiments/symbolic-propagation']
+          items: [
+            'experiments/overview',
+            'experiments/generic-pattern-matching',
+            { type: 'category',
+              label: 'Join mode',
+              items: ['experiments/join-mode/join-mode', 'experiments/join-mode/recursive-joins']
```

Comment:
Since join-mode now moves to a new directory, can you also add a redirect for it? Here’s an example PR that you can follow: https://github.com/returntocorp/semgrep-docs/pull/274/files